webplugin/js/app/kommunicate.js

```diff
@@ -501,10 +501,10 @@ $applozic.extend(true,Kommunicate,{
         };
     },
     /**
-     * @param {Boolean} hide 
+     * @param {Boolean} display 
      */
-    showKommunicateWidget: function(hide) {
+    displayKommunicateWidget: function(display) {
         var kommunicateIframe = parent.document.getElementById('kommunicate-widget-iframe');
-        hide ? kommunicateIframe.classList.remove("kommunicate-hide-custom-iframe") : kommunicateIframe.classList.add("kommunicate-hide-custom-iframe");
+        display ? kommunicateIframe.classList.remove("kommunicate-hide-custom-iframe") : kommunicateIframe.classList.add("kommunicate-hide-custom-iframe");
```

Comment:
@ab14bhardwaj if someone calls displayKommunicateWidget(false) two times and then calls displayKommunicateWidget(true) will it work?

Reply:
@devashishmamgain yes it will work because classList.add won't add same class twice even if you execute the line  "classList.add("kommunicate-hide-custom-iframe")". So even after n number of attempts you call "classList.add("kommunicate-hide-custom-iframe")" it will work and remove the class and show our widget.